core/src/test/java/org/apache/iceberg/avro/TestAvroNameMapping.java

```diff
@@ -369,15 +377,15 @@ protected Record writeAndRead(
     Record record = super.writeAndRead(desc, writeSchema, readSchema, inputRecord);
     Record projectedWithNameMapping =
         writeAndRead(writeSchema, readSchema, inputRecord, MappingUtil.create(writeSchema));
-    Assert.assertEquals(record, projectedWithNameMapping);
+    Assertions.assertThat(projectedWithNameMapping).isEqualTo(record);
     return record;
   }
 
   private Record writeAndRead(
       Schema writeSchema, Schema readSchema, Record record, NameMapping nameMapping)
       throws IOException {
 
-    File file = temp.newFile();
+    File file = new File(temp, "test.avro");
```

Comment:
should this be handled similarly to https://github.com/apache/iceberg/pull/8056/files#diff-d4debd1e3696fc5a7684ef1cf178c7d100aedfad26ffbca0560038eb98e90487R98?